src/priors/common.jl

```diff
@@ -1,20 +1,124 @@
-abstract type AbstractODEFilterPrior{elType} end
+"""
+    AbstractGaussMarkovProcess{elType}
+
+Abstract type for Gauss-Markov processes.
+
+Gauss-Markov processes are solutions to linear time-invariant stochastic differential
+equations (SDEs). Here we assume SDEs of the form
+```math
```

Comment:
Make this the same as the SDE in the docs! Then we also have the integrator structure at least somewhat documented.